types/unist/unist-tests.ts

```diff
@@ -1,56 +1,93 @@
-import { Data, Point, Position, Node, Literal, Parent } from 'unist';
+import { Point, Position, Node, Literal, Parent, NodeData } from 'unist';
 
-const data: Data = {
+const data = {
     string: 'string',
     number: 1,
     object: {
-        key: 'value'
+        key: 'value',
     },
     array: [],
     boolean: true,
-    null: null
+    null: null,
 };
 
 const point: Point = {
     line: 1,
     column: 1,
-    offset: 0
+    offset: 0,
 };
 
 const position: Position = {
     start: point,
     end: point,
-    indent: [1]
+    indent: [1],
 };
 
 const node: Node = {
     type: 'node',
     data,
-    position
+    position,
 };
 
-const text: Literal = {
+const text: Literal<string> = {
```

Comment:
I think it would be better to allow `Literal` here too — this change seems to imply `Literal` on its own doesn’t work anymore?

Reply:
It's allowed

Reply:
Could you then not change it?

Reply:
It is also used below. So maybe I should add a new one.